webui/react/src/services/apiConfig.ts

```diff
@@ -21,6 +21,7 @@ export const detApi = {
   Commands: new Api.CommandsApi(ApiConfig),
   Experiments: new Api.ExperimentsApi(ApiConfig),
   Internal: new Api.InternalApi(ApiConfig),
+  Jobs: new Api.JobsApi(ApiConfig),
```

Comment:
**chore:** I think `updateDetApi` below needs an update call for key `Jobs` when the `apiConfig` changes.

Reply:
good catch